api/src/pcapi/routes/backoffice_v3/user_generation/forms.py

```diff
@@ -1,9 +1,19 @@
+from pcapi.core.subscription.models import SubscriptionStep
 from pcapi.core.users import constants as users_constants
 from pcapi.routes.backoffice_v3.forms import fields
 from pcapi.routes.backoffice_v3.forms import utils
 
 
 class UserGeneratorForm(utils.PCForm):
     age = fields.PCIntegerField("Age", default=users_constants.ELIGIBILITY_AGE_18)
-    is_email_validated = fields.PCSwitchBooleanField("Email validé", default=True)
+    step = fields.PCSelectField(
+        "Étape de validation",
+        choices=[
+            (step.value, step.value.title().replace("-", " "))
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏Haha petit hack pour rendre ça lisible !
On peut faire une petite fonction `get_step_title` mais low logaf aussi (peut être demander aux devs BO3)

Reply:
Tu veux dire ajouter une fonction `get_step_title` à `SubscriptionStep` ? Ou ailleurs ?

Reply:
Ah yes pourquoi pas à `SubscriptionStep`
Mais sinon je pensais une petite fonction helper